metrics.go

```diff
@@ -82,6 +82,15 @@ type Metrics struct {
 
 	// Number of times baggage restrictions failed to update.
 	BaggageRestrictionsUpdateFailure metrics.Counter `metric:"baggage_restrictions_updates" tags:"result=err"`
+
+	// Number of times debug spans were throttled.
+	ThrottledDebugSpans metrics.Counter `metric:"throttled_debug_spans"`
```

Comment:
can you assert that these metrics are being used?

Reply:
Done